.ci/Manifest.toml

```diff
@@ -15,37 +15,29 @@ version = "0.2.0"
 uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
 [[BinaryBuilder]]
-deps = ["Base64", "Dates", "GitHub", "HTTP", "InteractiveUtils", "JLD2", "JSON", "LibGit2", "Libdl", "Logging", "LoggingExtras", "MbedTLS", "ObjectFile", "Pkg", "PkgLicenses", "ProgressMeter", "REPL", "Random", "Registrator", "RegistryTools", "SHA", "Sockets", "UUIDs", "ghr_jll"]
-git-tree-sha1 = "e953eddbb58fe1b0eb35c3e10cbb7f97862b43ee"
+deps = ["Base64", "CodecZlib", "Dates", "GitHub", "HTTP", "InteractiveUtils", "JLD2", "JSON", "LibGit2", "Libdl", "Logging", "LoggingExtras", "MbedTLS", "ObjectFile", "Pkg", "PkgLicenses", "ProgressMeter", "REPL", "Random", "Registrator", "RegistryTools", "SHA", "Sockets", "UUIDs", "ghr_jll"]
+git-tree-sha1 = "61862cec91f1ec721ad6309d2053d174263b4f91"
 repo-rev = "master"
 repo-url = "https://github.com/JuliaPackaging/BinaryBuilder.jl.git"
 uuid = "12aac903-9f7c-5d81-afc2-d9565ea332ae"
 version = "0.2.4"
 
-[[BinaryProvider]]
-deps = ["Libdl", "Logging", "Pkg", "SHA"]
-git-tree-sha1 = "a4c58d55ef3ad4c00523510917934ad2fb0ba194"
-repo-rev = "master"
-repo-url = "https://github.com/JuliaPackaging/BinaryProvider.jl.git"
-uuid = "b99e7846-7c00-51b0-8f62-c81ae34c0232"
-version = "0.6.0"
```

Comment:
We don't have anything else depending on BinaryProvider :smiling_imp:

Reply:
![](https://media1.giphy.com/media/MahDrOWLffiMKpiVV0/giphy.gif?cid=ecf05e47940cab2f9a655ef316a64cef0fdbe76b6a0dc89e&rid=giphy.gif)